nym-api/src/support/cli/run.rs

```diff
@@ -80,11 +81,24 @@ pub(crate) async fn execute(args: Args) -> anyhow::Result<()> {
             axum_shutdown.task_manager_mut().signal_shutdown().ok();
             axum_shutdown.task_manager_mut().wait_for_shutdown().await;
 
-            axum_shutdown.shutdown_axum();
+            let running_server = axum_shutdown.shutdown_axum();
+
+            match running_server.await {
+                Ok(server_res) => match server_res {
+                    Ok(_) => {
+                        info!("Axum HTTP server shut down without errors");
+                    }
+                    Err(err) => {
+                        error!("Axum HTTP server terminated with the error: {err}");
+                        anyhow::bail!(err)
+                    }
+                },
+                Err(err) => {
+                    error!("Server task panicked: {err}");
+                }
+            };
```

Comment:
suggestion: it's pretty neat to do 
```
match running_server.await {
   Ok(Ok(_)) => { .. }
   Ok(Err(err)) => {.. }
   Err(err) => { .. }
}
```
for situations like this to avoid nesting

Reply:
nice :ok_hand:  added